.ci_support/circle_.yaml

```diff
@@ -0,0 +1,2 @@
+zlib:
+- '1.2'
```

Comment:
Just trying to make sense of this, so we should be specifying `zlib`'s version when building `zlib` or is this some kind of bug? If not a bug, could someone please explain what this is doing in this case?

Reply:
This is probably an artifact.  It is almost certainly not used in the recipe.

Reply:
Is this something we should be fixing at the `conda-smithy` or `conda-build` level?

Reply:
Doesn't need a fix, IMHO.  Can be ignored.  If you must, probably conda-build.

Reply:
Perhaps it doesn't hurt the build currently, but am concerned it could in the future.

Also it's a bit confusing to see this as an end user. Chances if I'm confused, so will be a sizable portion of conda-forge's user base. Avoiding having to answer this same question 100s of times is already motivation enough to fix IMHO. :)

Have raised in issue ( https://github.com/conda/conda-build/issues/2701 ).